hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/command/AlterHoodieTableRenameCommand.scala

```diff
@@ -46,6 +46,17 @@ class AlterHoodieTableRenameCommand(
 
       // Call AlterTableRenameCommand#run to rename table in meta.
       super.run(sparkSession)
+
+      // update table properties: path
+      if (hoodieCatalogTable.catalogProperties.contains("path")) {
+        val catalogTable = sparkSession.sessionState.catalog.getTableMetadata(newName)
+        if (catalogTable.tableType == CatalogTableType.MANAGED) {
```

Comment:
The non `MANAGED` table gives a new path, why can't it be modified?

Reply:
just consistent with metastore design logic.  For **MANGED** table, ms will update storage location automatically when rename table, but **EXTERNAL** will not

Reply:
> just consistent with metastore design logic. For **MANGED** table, ms will update storage location automatically when rename table, but **EXTERNAL** will not

Does not automatically update, do HMS synchronization API modification, catalog table refresh can also be satisfied?

Reply:
my mistake, the managed table which has be set location alson will not update sds, there is no relationship with managed and external table.